pyre/bench/synth/a_raise_through_the_recursive_portal_reaches_its_handler.py

```
@@ -0,0 +1,92 @@
+
+
+def hot(k):
+    caught = 0
+    wrong = 0
+    seen = []
+    for _ in range(k):
+        try:
+            rec(DEPTH, INNER)
+        except ValueError:
+            caught += 1
+        except BaseException as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `BaseException`

(BLE001)